docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -130,31 +130,31 @@ A sample supervisor spec is shown below:
+| `reportParseExceptions`           | Boolean        | *DEPRECATED*. If true, exceptions encountered during parsing will be thrown and will halt ingestion; if false, unparseable rows and fields will be skipped. Setting `reportParseExceptions` to true will override existing configurations for `maxParseExceptions` and `maxSavedParseExceptions`, setting `maxParseExceptions` to 0 and limiting `maxSavedParseExceptions` to no more than 1.                                                                                                                                                                                                                                                       | no (default == false)                                                                                        |
+| `handoffConditionTimeout`         | Long           | Milliseconds to wait for segment handoff. It must be >= 0, where 0 means to wait forever.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | no (default == 0)                                                                                            |
+| `resetOffsetAutomatically`        | Boolean        | This option _MUST_ not be set if you don't want to ignore missing data from Kafka. Once this is set, the supervisor _automatically_ resets the consumer offset for each Kafka partitions whenever the next offset it's trying to fetch is less than the earliest available offset in Kafka. The consumer offset will be reset to either the earliest or latest offset available in Kafka depending on `useEarliestOffset` property of `KafkaSupervisorIOConfig`. Please note that this could lead _data loss_ (when `useEarliestOffset` = false) or _data duplicate_ (when `useEarliestOffset` = true). The missing offsets in Kafka typically occurs when messages in Kafka are no longer available for consumption and therefore won’t be ingested into Druid. If set to false then ingestion for that particular partition will halt and manual intervention is required to correct the situation, please see [Reset Supervisor API](../../operations/api-reference.html#supervisors). | no (default == false) |
```

Comment:
Typos:
- for each Kafka partitions -> for each Kafka partition
- depending on `useEarliestOffset` -> depending on the `useEarliestOffset`
- could lead _data loss_ -> could lead to _data loss_
- missing offsets in Kafka typically occurs -> missing offsets in Kafka typically occur

Reply:
Thank you for the review! The whole description is rewritten. Please take another look.